front-end/src/components/UserForm/SignIn.tsx

```diff
@@ -7,26 +7,53 @@ interface UserForm {
 interface State {
```

Comment:
Im ok if you want to go forward with State and not Hooks but I thought that was the way we were going to to do it